ControlLaptop/Examples/Log-windtunnel-influx/log-windtunnel-influx.py

```diff
@@ -0,0 +1,170 @@
+# Example for logging data to influxdb using python
+# Sourced from the influxdb python docs
+
+import random
+import time
+from datetime import datetime 
+from influxdb import InfluxDBClient
+
+# Example data payload
+# Tags contain metadata about the measurement - always a string
+# Thinking we could have something like below for each measurement
+
+def main():
+        
+    telemetry = [
+           {
+            "measurement": "temperature",
+            "tags": {
+                "host": "server01",
+                "region": "us-west"
+            },
+            #"time": f"{datetime.now()}",
+            "fields": {
+                "Celsius": 1.0,
+                "Fahrenheit": 3,
+               }
+        },
+        
+          {
+            "measurement": "pressure",
+            "tags": {
+                "host": "server01",
+                "region": "us-west",
+                "config": "1",
+            },
+            "fields": {
+                "millibar": 1.0,
+                "Pascal": 3.0,
+                }
+         },
+        
+        
```

Comment:
There isn't really a standard for json, but consistency is good. For example, you have two lines here, but you have 1 line in my above comment.